FoodQuartet-Kit/UI Modules/Slot/SlotViewController.swift

```diff
@@ -314,4 +314,41 @@ extension SlotViewController {
         )
         return cell
     }
+    
+    private func customizeBarAttributes(for navigationItem: UINavigationItem) {
+        let size = 19.5
+        let mergedAttributes = NSMutableAttributedString()
+        
+        let characters = [
+            ("F", UIColor.calculateColor(.springHex)),
+            ("o", UIColor.calculateColor(.summerHex)),
+            ("o", UIColor.calculateColor(.fallHex)),
+            ("d", UIColor.calculateColor(.winterHex))
+        ]
+        for (char, color) in characters {
+            let currentChar = configureAttributes(for: char, color: color, fontSize: size, weight: .semibold)
+            mergedAttributes.append(currentChar)
+        }
+        
+        let additionalString = configureAttributes(for: "Quartet", color: .label, fontSize: size, weight: .regular)
+        mergedAttributes.append(additionalString)
+        
+        let titleLabel = UILabel()
+        titleLabel.attributedText = mergedAttributes
+        
+        navigationItem.titleView = titleLabel
+        navigationItem.backButtonTitle = K.L10n.back
+    }
+    
+    
+    // MARK: - UI Layout Helper
+    
+    private func configureAttributes(for content: String, color: UIColor, fontSize: CGFloat, weight: UIFont.Weight) -> NSAttributedString {
+        let style: [NSAttributedString.Key: Any] = [
+            .foregroundColor: color,
+            .font: UIFont.systemFont(ofSize: fontSize, weight: weight)
+        ]
+        let attributed = NSAttributedString(string: content, attributes: style)
+        return attributed
+    }
```

Comment:
[![imo-badge](https://img.shields.io/badge/review-imo-orange.svg)]()  
Replace `NSAttributedString` with `AttributedString` to enhance flexibility and readability:  

- While `NSAttributedString` is an imperative approach and might lead to increased complexity, `AttributedString` follows a declarative approach with type safety.